common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -1596,6 +1598,66 @@ colored_emblem_texture_lists = {
 			2000 	= "ce_ogre11.dds"
 			2000 	= "ce_ogre12.dds"
 		}
+		### Harpy
+		special_selection = {
+			trigger = { scope:culture = { has_graphical_culture = harpy_coa_gfx } }		
+			2000    = "ce_wings.dds"
+			2000    = "ce_vol.dds"
+			2000    = "ce_demi_vol.dds"
+			2000    = "ce_owl.dds"
+			2000    = "ce_norse_bird_alstad.dds"
+			2000    = "ce_martlet.dds"
+			2000    = "ce_falcon.dds"
+			2000    = "ce_eagle_claw.dds"
+			2000    = "ce_eagle_head.dds"
+		}
+		### Dryad and Night Elf (Druidic)
+		special_selection = {
+			trigger = { scope:culture = { has_graphical_culture = druidic_coa_gfx } }
+			2000    = "ce_antelope.dds"
+			2000    = "ce_antler.dds"
+			2000    = "ce_antler_pale.dds"
+			2000    = "ce_antlers_attire.dds"
+			2000    = "ce_attire.dds"
+			2000    = "ce_bear_claw.dds"
+			2000    = "ce_boar_passant_02.dds"
+			2000    = "ce_bugle_horn_random.dds"
+			2000    = "ce_chicots_saltire.dds"
+			2000    = "ce_demi_vol.dds"
+			2000    = "ce_eagle_claw.dds"
+			2000    = "ce_eagle_head.dds"
+			2000    = "ce_goat.dds"
+			2000    = "ce_grape_tree.dds"
+			2000    = "ce_horn.dds"
+			2000    = "ce_horse_rampant.dds"
+			2000    = "ce_horse_statant.dds"
+			2000    = "ce_owl.dds"
+			2000    = "ce_panther.dds"
+			2000    = "ce_peacock_01.dds"
+			2000    = "ce_religion_magyar.dds"
+			2000    = "ce_religion_waaqism.dds"
+			2000    = "ce_squirrel.dds"
+			2000    = "ce_swan.dds"
+			2000    = "ce_tree.dds"
+			2000    = "ce_wolf.dds"
+			2000    = "ce_wolf_head.dds"
+			2000    = "ce_wings.dds"
+			2000    = "ce_vol.dds"
+			2000    = "ce_religion_custom_faith_6.dds"
+			2000    = "ce_unicorn.dds"
+			
+				### Ancient
+				special_selection = {
+				trigger = { scope:culture = { has_graphical_culture = ancient_coa_gfx } }
+				4000    = "ce_yggdrasil.dds"
+				4000    = "ce_yggdrasil_ockelbo.dds"
+				4000    = "ce_tree.dds"
+				4000    = "ce_religion_waaqism.dds"
+				4000    = "ce_religion_custom_faith_6.dds"
+				4000    = "ce_oak_leaf.dds"
+				4000    = "ce_laurels.dds"
+			}
```

Comment:
You put the Ancient block inside the Druidic block.
```suggestion
		special_selection = {
			trigger = { scope:culture = { has_graphical_culture = druidic_coa_gfx } }
			2000    = "ce_antelope.dds"
			2000    = "ce_antler.dds"
			2000    = "ce_antler_pale.dds"
			2000    = "ce_antlers_attire.dds"
			2000    = "ce_attire.dds"
			2000    = "ce_bear_claw.dds"
			2000    = "ce_boar_passant_02.dds"
			2000    = "ce_bugle_horn_random.dds"
			2000    = "ce_chicots_saltire.dds"
			2000    = "ce_demi_vol.dds"
			2000    = "ce_eagle_claw.dds"
			2000    = "ce_eagle_head.dds"
			2000    = "ce_goat.dds"
			2000    = "ce_grape_tree.dds"
			2000    = "ce_horn.dds"
			2000    = "ce_horse_rampant.dds"
			2000    = "ce_horse_statant.dds"
			2000    = "ce_owl.dds"
			2000    = "ce_panther.dds"
			2000    = "ce_peacock_01.dds"
			2000    = "ce_religion_magyar.dds"
			2000    = "ce_religion_waaqism.dds"
			2000    = "ce_squirrel.dds"
			2000    = "ce_swan.dds"
			2000    = "ce_tree.dds"
			2000    = "ce_wolf.dds"
			2000    = "ce_wolf_head.dds"
			2000    = "ce_wings.dds"
			2000    = "ce_vol.dds"
			2000    = "ce_religion_custom_faith_6.dds"
			2000    = "ce_unicorn.dds"
		}
			
		### Ancient
		special_selection = {
			trigger = { scope:culture = { has_graphical_culture = ancient_coa_gfx } }
			4000    = "ce_yggdrasil.dds"
			4000    = "ce_yggdrasil_ockelbo.dds"
			4000    = "ce_tree.dds"
				4000    = "ce_religion_waaqism.dds"
			4000    = "ce_religion_custom_faith_6.dds"
			4000    = "ce_oak_leaf.dds"
			4000    = "ce_laurels.dds"
		}
```

Reply:
This was intentional. This way the Ancients would also use druidic_coa_gfx CoAs, although not as frequently. Does the code work?

Reply:
Nope. It means that the ancients have to have both `ancient_coa_gfx` and `druidic_coa_gfx`. Cultures that have only `ancient_coa_gfx` won't have ancient nor druidic CoAs.

From your explanation, you need to add `ancient_coa_gfx` in the druidic block.
Get the ancient block outside of the druidic block.